cli/status/lua/instance_state.lua

```diff
@@ -0,0 +1,57 @@
+local ok, config = pcall(require, 'config')
+if ok and config ~= nil and type(config) == "table" and type(config.info) == "function" then
+    config = config:info()
```

Comment:
Nit: The `config` variable stores the module first, then the info result. Then `config_info` stores a shrinked info result. I would name it as `config`, `config_info_full` and `config_info` or somehow like this. I don't like that we name the info result as `config`.